lib/jsonld.js

```diff
@@ -1005,16 +1003,6 @@ jsonld.RequestQueue = require('./RequestQueue');
 /* WebIDL API */
 jsonld.JsonLdProcessor = require('./JsonLdProcessor')(jsonld);
 
-// setup browser global JsonLdProcessor
-if(_browser && typeof global.JsonLdProcessor === 'undefined') {
```

Comment:
Can this just be changed to `globalThis` to avoid a breaking change?

Reply:
Easiest thing to do would be to change `global` to `window`. Would you prefer that?

Reply:
`globalThis` exists for this purpose, let's use it.  That was the plan in my head, it just hadn't happened yet.  Now with the node >=12 req, and modern bundler tools, it should be easier than before.  Using `window` won't work in web workers that just have `self`.

Reply:
Also on the TODO list was to break out node/browser code using modern practices (that detection code is ancient).  I think it's an easy change here if that makes a difference for whatever problem this code was causing.